operator/src/main/java/org/bf2/operator/events/ResourceEventSource.java

```diff
@@ -45,7 +45,7 @@ protected void handleEvent(HasMetadata resource, ResourceAction action) {
         // the operator may not have inited yet
         if (getEventHandler() != null) {
             ResourceID.fromFirstOwnerReference(resource).ifPresentOrElse(
-                    ownerId -> getEventHandler().handleEvent(new ResourceEvent(action, ownerId)),
```

Comment:
@MikeEdgar  this is the only change that I haven't been able to resolve. Can you help me with this?

Reply:
Here you can pass `null` as the third parameter.